tests/functional/t/typevar_naming_style_default.txt

```diff
@@ -1,6 +1,10 @@
-typevar-name-incorrect-variance:11:0:11:21::"Type variable ""GoodNameWithoutContra"" is contravariant, use ""GoodNameWithoutContra_contra"" instead":INFERENCE
-typevar-name-incorrect-variance:19:0:19:1::"Type variable ""T"" is covariant, use ""T_co"" instead":INFERENCE
-typevar-name-incorrect-variance:24:0:24:8::"Type variable ""T_contra"" is covariant, use ""T_co"" instead":INFERENCE
+typevar-name-incorrect-variance:11:0:11:21::"Type variable name does not reflect variance. ""GoodNameWithoutContra"" is contravariant, use ""GoodNameWithoutContra_contra"" instead":INFERENCE
+typevar-double-variance:19:0:19:1::TypeVar cannot be both covariant and contravariant:INFERENCE
+typevar-name-incorrect-variance:19:0:19:1::Type variable name does not reflect variance%s:INFERENCE
+typevar-double-variance:23:0:23:4::TypeVar cannot be both covariant and contravariant:INFERENCE
+typevar-name-incorrect-variance:23:0:23:4::Type variable name does not reflect variance%s:INFERENCE
```

Comment:
The %s here doesn't look correct? Can we remove it?

Reply:
Good spot, that should be filled by an empty string so let me take a look at what's going on there!

Reply:
Thanks for that fix. Let's see if that crashes any other tests 😄